### ISSUE TITLE
include PV data from _both_ PassivSystems and PVOutput.org

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/nowcasting_dataset#554]
## Detailed Description
my hunch from the last couple of weeks of ML experiments is that the lowest hanging fruit for improving our PV nowcasts further might be to extract as much info as we can from the recent history of PV data... so it might be nice to combine the PassivSystems PV data with the PVOutput.org data, to get our hands on as much PV data as possible.

(This might also give us more resilience in production... if the PassivSystems data goes down for a while, hopefully we can still use the PVOutput.org data.  And visa-versa).

**Comment by peterdudfield** [comment]
I think this would be great. 

For ML would you combine both power data and metadata into one file each, or have it so that [PVDataSource](https://github.com/openclimatefix/nowcasting_dataset/blob/main/nowcasting_dataset/data_sources/pv/pv_data_source.py#L28) can load both? I think combining would be a good idea.

**Comment by JackKelly** [comment]
> I think combining would be a good idea.

I agree!  I would vote for `nowcasting_dataset` doing the hard work of combing the Passiv and PVOutput.org datasets, so `nowcasting_dataloader` can sit back and relax and just load PV data from a single set of NetCDF files :slightly_smiling_face:

**Comment by peterdudfield** [comment]
or even before? See how easy it is to merge both metadata files, and both power files in a script to make a new version of the combined data. 

pros:
- keeps nowcasting_dataset simplt
cons: 
- if we update pv data, then we to two this extra step

Perhaps we have an option of `PVDataSource` taking into a list of files, and we just have to make sure they are in the same format, i.e same data variables (and columns names in the metadata)

**Comment by JackKelly** [comment]
sounds good... as long as we can create unique PV system IDs for each PV system in the combined dataset...
